chapter_appendix-mathematics-for-deep-learning/statistics.md

```diff
@@ -229,9 +229,9 @@ To complete the story of how to work with hypothesis testing, we need to now int
 
 The *statistical significance* measures the probability of erroneously rejecting the null hypothesis, $H_0$, when it should not be rejected, i.e.,
 
-$$ \text{statistical significance }= 1 - \alpha = P(\text{reject } H_0 \mid H_0 \text{ is true} ).$$
+$$ \text{statistical significance }= 1 - \alpha = 1 - P(\text{reject } H_0 \mid H_0 \text{ is true} ).$$
```

Comment:
This is the definition of statistical significance. I don't think it is a mistake. Can you change it back?

Reply:
P(\text{reject } H_0 \mid H_0 \text{ is true} is eaqual to the alpha.
https://en.wikipedia.org/wiki/Statistical_significance
>More precisely, a study's defined significance level, denoted by {\displaystyle \alpha }\alpha , is the probability of the study rejecting the null hypothesis, given that the null hypothesis was assumed to be true;